.github/workflows/paralleltest.yml

```diff
@@ -17,6 +17,13 @@ on:
       system:
         description: System being tested
         required: true
+      version:
+        description: JDK version
+        required: false
+        type: choice
+        options:
```

Comment:
Should there be an option for `jdk21` now? Or does that require additional development?